tckprocess.adoc

```diff
@@ -0,0 +1,422 @@
+# MicroProfile TCK Process 1.0
+
+Specification Teams under the Eclipse Foundation Specification
+Process MUST produce a Technology Compatibility Kit (TCK) that delivers
+on the promise of enabling multiple compatible implementations.
+
+This document defines:
+
+-   Materials a TCK MUST possess to be considered suitable for
+    delivering portability
+
+-   Process for challenging tests and how these challenges are resolved
+
+-   Means of excluding released TCK tests from certification
+    requirements
+
+-   Policy on improving TCK tests for released specifications
+
+-   Process for self-certification
+
+[#_materials_for_a_tck_release]
```

Comment:
Since in MP, TCK relase happens together with the api/spec. I suggest not to call this out. Therefore, I suggest to remove this section.

Reply:
@dblevins please add a section about the version of TCK being the same as APIs/Spec